phat/contracts/brick_profile/lib.rs

```diff
@@ -255,12 +276,9 @@ mod brick_profile {
             Ok(id)
         }
 
-        /// Dump an EVM account secret key, this will disable the account and zeroize the sk, only owner is allowed
+        /// Dump an EVM account, this will disable the account, only owner is allowed
         #[ink(message)]
         pub fn dump_evm_account(&mut self, id: ExternalAccountId) -> Result<()> {
-            // Deprecated in first release
-            return Err(Error::Deprecated);
```

Comment:
Not sure what's the purpose of this operation.
Security Caution: the state could be rollback by restore the entire pruntime to a history checkpoint.